client/server/api/sign-in-with-apple.js

```diff
@@ -65,6 +67,12 @@ function redirectToCalypso( request, response, next ) {
 		client_id: config( 'apple_oauth_client_id' ),
 		state: state.oauth2State,
 	} );
+
+	const isRelativeUrl = originalUrlPath.startsWith( '/' ) && ! originalUrlPath.startsWith( '//' );
```

Comment:
I'm curious why we didn't have this logic before? Is `originalUrlPath` absolute or relative before this change? 🤔